vignettes/data-transform-as-shiny-module.Rmd

```diff
@@ -164,76 +164,3 @@ if (interactive()) {
   shinyApp(app$ui, app$server)
 }
 ```
-
-## Custom placement of the transform UI
-
-When a custom transformation is used, the UI for the transformation is placed below the filter panel.
-However, there is a way to customize the placement of the UI inside the module content.
-
-In order to place the transformation UI inside the module there are few things one has to do:
-1. Create a custom module wrapper function.
-2. Call the desired module in the module wrapper function and store it in a variable so it's UI can be modified.
-3. Modify the UI of the module with the transform UI at the desired location by calling the `ui_transform_data`. Note that in order for the transform to work you need to change the namespace of the `id` by passing `NS(gsub("-module$", "", id), "data_transform")`.
-4. Set the `custom_ui` attribute of the `module$transformers` to `TRUE`.
-
-Now the custom module should embed the transformation UI inside the module content.
-
-Here is an example of a custom module wrapper function that modifies the `example_module` module.
-```{r}
-example_module_encoding <- function(label = "example module (on encoding)", datanames = "all", transformers = list()) {
-  mod <- example_module(label, datanames, transformers)
-  mod$ui <- function(id) {
```

Comment:
This was super-hacky solution. Avoiding public api and overwriting module's ui.

Reply:
And it will be competely redundant with `decorator` work :+1: